### DISCUSSION TITLE
DJI Mic 2 and Hero 13

### DISCUSSION THREAD

**Discussion by TJ311** [gopro/labs#1063] [General]
I installed the Hero 13 latest LABS firmware and the DJI mic 2 is not able to pair with Hero 13. When I do the pair procedure, the mic does not show up in Hero 13 Bluetooth device list. (Hero 13 in Pair mode and searching - DJI Mic in pair mode). Anyone else having this issue?

**Comment by dnewman-gpsw** [comment]
![image](https://github.com/user-attachments/assets/3d09e915-e14a-4854-a000-2f9db6c2b012)
It works fine for me.  I just tried it with Labs on HERO13.  Make sure you have the Mic2 in BLE mode (blue) and linking (rapid blinking.)

**Reply by TJ311** [reply]
Yes it's in pairing mode and the blue light is blinking fast. Hmmmm. I just paired my remote control and had no issue. I'll try my other DJI transmitter

**Reply by dnewman-gpsw** [reply]
I did have to select Mic2 on the devices pairing screen, whereas the Remote was automatic.

**Reply by TJ311** [reply]
Really strange. It kind of shows up randomly. I was able to select it one time from the device list but it then went to "unable to connect". Removed the paired device that wouldn't connect and tried again to pair the mic 2. And once again, the Hero 13 doesn't see the Mic 2. Both in pairing mode. I tried resetting the bluetooth / wifi connections in the gopro. No help. 

![20240912_134704](https://github.com/user-attachments/assets/f49dedd3-de81-419b-81c9-82963ba1c775)

**Reply by dnewman-gpsw** [reply]
This is not a GoPro Labs issue, as it doesn't alter BLE support in any way.  You might have to try GoPro support (or DJI support.) My only experience was just above, I only received the Mic2 yesterday as a user like you, yet it worked for me.

**Reply by TJ311** [reply]
Thanks. Appreciate the response.

**Reply by MtnXfreeride** [reply]
Just to provide some more numbers here.  I have THREE Hero 13s and ONLY 1 of the 3 sees my DJI mic 2.. the other 2 refuse to show it when pairing.

**Reply by TJ311** [reply]
Oh that isn't good. I've been working on mine. I have a second hero 13 and it paired right away and right after that..... the original Hero 13 miraculously paired. Sad to say that neither of these two cameras stay connected though. Both lose connection after a minute or two of recording. So.... I have a third Hero 13 arriving next week. I'll keep my fingers crossed that I'll get lucky on one of them like you did. I use 3 cameras simultaneously and then use multi cam editing for my YouTube fishing channel. I currently use a DJI Osmo Action 4 as my 4th camera and it's pretty much just for filming my intros. Was looking forward to getting rid of that camera and using one of my 13's. Fingers crossed! 

Just a little more troubleshooting I did..... I returned the hero 13 back to stock firmware and did a factory reset. It made no difference for the Mic 2 pairing issues. I also factory reset the Mic 2. The firmware is the latest from DJI. You and I certainly can't be the only unlucky folks.

**Reply by dnewman-gpsw** [reply]
In my testing, the Mic2 needs be turned on after the HERO13 for the previous pairing to work. Once it is paired it works on three cameras I have tried.  I did a 15 minute test, without any dropouts.  Nothing Labs related (I was using Labs firmware), but my experience with Mic2.

**Reply by TJ311** [reply]
Thanks for the info. I'll give that a try! and report back.

**Comment by TJ311** [comment]
Just an update. I am still NOT able to get DJI Mic 2 connected to my Hero 13 #1.  I've tried it by turning on the gopro first, then the mic. And I've tried it the other way around. Nothing seems to work.  The Mic 2 does not show up in the device list when in pairing mode. Once in a great while, it'll show up for a split second, then lose connection. 

I have a second Hero 13 #2 camera and that one connects perfectly and stays connected / reconnects perfectly too. 

 I'm really puzzled as to why some cameras work perfectly and others not. 

![Untitled video](https://github.com/user-attachments/assets/40e3941d-67fd-4972-b14e-706dc8610c8c)
